front/app/containers/Admin/projects/project/information/ReportTab/messages.ts

```diff
@@ -43,4 +43,24 @@ export default defineMessages({
     id: 'app.containers.Admin.projects.project.information.ReportTab.reportVisible',
     defaultMessage: 'Report visible to public',
   },
+  notVisibleNotStarted: {
+    id: 'app.containers.Admin.projects.project.information.ReportTab.notVisibleNotStarted',
+    defaultMessage:
+      'This report is not visible to the public. To make it publicly visible once the phase starts, enable the "Report visible to public" toggle.',
```

Comment:
I believe these 4 copies are pretty good, but maybe they still could be improved a bit. Not 100% sure about all ideas, but here are some of them.
1. The same idea of "public" is expressed in 3 different ways: visible to the public, publicly visible, publicly available. 
2. Could we just always use "public"? Including the toggle label, so it'll be "Public" (w/o report).
3. The copies are a bit too long and repetitive. E.g., this one could be `This report is not public. To make it public once the phase starts, enable the "Public" toggle.`
4. Maybe we don't need banners for all 4 cases. Maybe when the toggle is off, we don't need to show the banner if the phase has not started - it's clear enough it's not public. And when the toggle is on, we don't need to show the banner if the phase has started - it's also clear it's public. So, maybe we only need a banner in 2 cases instead of 4.